requirements/base.in

```diff
@@ -24,5 +24,6 @@ setuptools
 sqlalchemy
 sqlalchemy-migrate
 statsd
+typing-extensions
```

Comment:
Why is this necessary?

Reply:
Uhh, whoops. It's not! (I was experimenting with adding types when I noticed this - clearly this got committed to the wrong branch.)